### ISSUE TITLE
Docs don't indicate that methods can use types directly

### ISSUE THREAD

**Issue by fritzy** [fritzy/ape-ecs#25]
Documentation needs to be updated to show that methods that type a type-name string can also take the type constructor itself, directly.

**Comment by vmwxiong** [comment]
Is this true of `createEntity`?

Trying to figure out a way to call `createEntity` and pass in a list of components types without using strings anywhere.

**Comment by fritzy** [comment]
Yeah, that's true... it expects a definition object. I meant that anytime you use a Type, where in that case you're mentioning an instance. You can use anything other than strings as object keys.
